.github/workflows/docs_test.yml

```diff
@@ -0,0 +1,55 @@
+name: docs_test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: docs_test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  GO_VERSION: 1.14.2
+
+jobs:
+  docs_test:
+    name: docs_test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Caching dependency
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+            ~/go/bin
+          key: ${{ runner.os }}-v1-go-mod-${{ hashFiles('**/go.mod') }}-${{ hashFiles('**/go.sum') }}
+
+      - name: go mod download
+        run: |
+          [[ -d /go/pkg/mod/cache ]] && exit 0
```

Comment:
```suggestion
          [[ -d ~/go/pkg/mod ]] && exit 0
```

Reply:
I have made the above changes as per your suggestion in gotests, docs_test, and modules_and_generate jobs.
From this now my understanding is for ubuntu instead of checking the cash directory (`/go/pkg/mod/cache`) we need to check the go module path (`~/go/pkg/mod`). Correct me if I am wrong.

Reply:
We just need to check for the default path set by `actions/setup-go`.